packages/x-charts/src/internals/getGraphemeCount.ts

```diff
@@ -0,0 +1,24 @@
+const segmenter =
+  typeof window !== 'undefined' && 'Intl' in window && 'Segmenter' in Intl
+    ? new Intl.Segmenter(undefined, { granularity: 'grapheme' })
+    : null;
+
+function getGraphemeCountFallback(text: string) {
+  return text.length;
+}
+
+function getGraphemeCountModern(text: string) {
+  const segments = segmenter!.segment(text);
+
+  let count = 0;
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/naming-convention,no-underscore-dangle
```

Comment:
Need to disable all of these, otherwise CI will fail 🤣 

`segments` is an iterator, so we need to iterate it to count. I could do `Array.from()` but didn't want to allocate an array just to obtain its length. 

As such, the `_unused` var is useless, but I can't name it `_` or `unused` because that will cause different lint errors to fire. 

<img width="1442" alt="image" src="https://github.com/user-attachments/assets/e6e977fd-fbb6-40e9-93f8-752d41d3ecff" />